lib/main.dart

```diff
@@ -178,6 +178,14 @@ class KAGAppState extends State<KAGApp> with TickerProviderStateMixin {
     });
   }
 
+  int getTabControllerIndex() {
+    if (kIsWeb) {
+      if (KAGAppState.app.appType == AppType.MOBILE_SITE) return 2;
+      return 1;
+    }
+    return 0;
+  }
+
```

Comment:
Warum?
Können wir nicht Datenschutz einfach nicht als ersten Tab nehmen?
Das Problem ist, dass sowas sehr schnell unüberschaubaren Code produziert der später komische Bugs verursacht

Reply:
Ne @themathman2 meint das muss so. Mir wäre das völlig egal wo das Icon in der Navbar platziert ist.

Reply:
@themathman2 können wir das nicht einfach ans Ende stellen?

Reply:
Mir egal ganz links oder ganz rechts. Nur nicht irgendwo mitten drin

Reply:
Hätten sie mal früher sagen können